handsontable/types/core.d.ts

```diff
@@ -59,7 +59,7 @@ export default class Core {
   getCell(row: number, column: number, topmost?: boolean): HTMLTableCellElement | null;
   getCellEditor(cellMeta: CellMeta): BaseEditor;
   getCellEditor(row: number, column: number): BaseEditor;
-  getCellMeta(row: number, column: number): CellProperties;
+  getCellMeta(row: number, column: number, options?: { skipMetaExtension: boolean }): CellProperties;
```

Comment:
Can I ask you to cover that typing with a test... I think this [test file](https://github.com/handsontable/handsontable/blob/develop/handsontable/src/__tests__/core/methods.types.ts#L58) needs to be updated.

Reply:
Done. Didn't know we have typing tests ;)